### ISSUE TITLE
ChemSpider API connection broken

### ISSUE THREAD

**Issue by pkrog** [pkrog/biodb#259]

**Comment by pkrog** [comment]
```
API broken: "XML content does not seem to be XML: Unauthorized web service usage. Please request access to this service."
```

**Comment by pkrog** [comment]
Solved by #292 and #293 .
ChemSpider has new paid plans. The free plan is limited to 1000 requests per month. For this reason ChemSpider tests in Travis-CI are now disabled. ChemSpider will only be tested on local computers, manually.
